### ISSUE TITLE
how to use --ignore?

### ISSUE THREAD

**Issue by jchannon** [monochromegane/the_platinum_searcher#118]
i am using this `pt --ignore-case --ignore=["*.html","*.js"] lib/`but its not ignoring those files with those extensions. am i doing something wrong?

**Comment by syohex** [comment]
I suppose `go-gitignore` has bug about glob pattern(https://github.com/monochromegane/go-gitignore/pull/1). So `--ignore` option does not work well now.

Original code works as below. `--ignore` options are specified, but `*.go` files and `*.json` file are matched.

```
% cd $GOPATH/src/github.com/monochromegane/the_platinum_searcher
% pt --ignore='*.go' --ignore='*.json' -l github
cmd/pt/main.go
find.go
option.go
platinum_searcher.go
formatter.go
ignore.go
wercker.yml
Godeps/Godeps.json
README.md
```

Original code with the patch(https://github.com/monochromegane/go-gitignore/pull/1) works as below. Result is expected.

```
% cd $GOPATH/src/github.com/monochromegane/the_platinum_searcher
% pt --ignore='*.go' --ignore='*.json' -l github
README.md
wercker.yml
```

**Comment by jchannon** [comment]
So I guess we need to wait for that patch to be fixed first?

**Comment by syohex** [comment]
> So I guess we need to wait for that patch to be fixed first?

Yes.

**Comment by jchannon** [comment]
Cool , thanks!

**Comment by monochromegane** [comment]
@syohex Thank you for the PR :smiley: 
I have merged now.
So you can try after `go get -u github.com/monochromegane/go-gitignore` and `go build`.

I will release new binary after this immediately.

**Comment by monochromegane** [comment]
@jchannon @syohex I have released v2.0.3 which contains this bug fix.
https://github.com/monochromegane/the_platinum_searcher/releases/tag/v2.0.3

Thanks for reporting and fixing a bug :smiley:

**Comment by jchannon** [comment]
Works a treat. Thanks guys!
